linuxdeploy-plugin-gtk.sh

```diff
@@ -124,12 +129,8 @@ echo "Copying more libraries"
 gobject_libdir="$("$PKG_CONFIG" --variable=libdir gobject-2.0)"
 gio_libdir="$("$PKG_CONFIG" --variable=libdir gio-2.0)"
 librsvg_libdir="$("$PKG_CONFIG" --variable=libdir librsvg-2.0)"
-cp $verbose \
-    "$gdk_libdir/"libgdk_pixbuf*.so* \
-    "$gobject_libdir/"libgobject*.so* \
-    "$gio_libdir/"libgio*.so* \
-    "$librsvg_libdir/"librsvg*.so* \
-    "$APPDIR/usr/lib/"
-cat >> "$HOOKFILE" <<EOF
-export LD_LIBRARY_PATH="\$APPDIR/usr/lib:\$LD_LIBRARY_PATH"
-EOF
+"$LINUXDEPLOY" --appdir="$APPDIR"                                           \
+	--library="$(find "$gdk_libdir"     -type f -name libgdk_pixbuf-*.so*)" \
```

Comment:
Thinking about it, with this syntax, you should probably add a `| head -n1` to make sure you process one hit per piece.

Alternatively, some bash list processing would do, too.